feature/movie-detail/src/main/java/io/filmtime/feature/movie/detail/MovieDetailScreen.kt

```diff
@@ -109,7 +157,7 @@ fun MovieDetailContent(
 
     Text(
       modifier = Modifier.padding(horizontal = 16.dp),
-      style = MaterialTheme.typography.titleLarge,
+      style = typography.titleLarge,
```

Comment:
As reviewer I can't easily find out where did this `typography` come from. It is not a common behavior to import it. At least I have not seen it before 😄